conf/modules/video_thread.xml

```diff
@@ -3,11 +3,10 @@
 <module name="video_thread" dir="computer_vision">
   <doc>
     <description>
-      Read video in a thread.
```

Comment:
First line of the description (until the dot) is used as brief description and displayed in http://docs.paparazziuav.org/latest/onboard_modules.html